Sources/SyrupCore/Generator/Renderer.swift

```diff
@@ -136,27 +136,27 @@ open class Renderer {
         return rendered
     }
     
-    func renderSubscription(subscription: IntermediateRepresentation.OperationDefinition, intermediateRepresentation: IntermediateRepresentation, subscriptionSelections: SelectionSetVisitor.Operation) throws -> String {
-        let name = "\(subscription.name)Subscription"
-        let queryString = intermediateRepresentation.fullQueryString(for: subscription).removingLeadingSpaces.removingNewLines
-        let importEnums = intermediateRepresentation.referencedEnums.isEmpty == false
-        let importInputs = intermediateRepresentation.referencedInputTypes.isEmpty == false
-        let importFragments = intermediateRepresentation.fragmentDefinitions.isEmpty == false
-        let requiresCustomEncoder = subscription.variables.contains(where: { $0.type.nestedScalar() is IntermediateRepresentation.CustomCodedScalar })
-        let context: [String: Any] = [
-            "name": name,
-            "operation": subscription,
-            "queryString": queryString,
-            "moduleName": config.project.moduleName,
-            "isQuery": true,
-            "importEnums": importEnums,
-            "importInputs": importInputs,
-            "importFragments": importFragments,
-            "requiresCustomEncoder": requiresCustomEncoder,
-            "selections": subscriptionSelections.selectionSet
-        ]
-        return try render(template: "Operation", asFile: true, context: context)
-    }
+	func renderSubscription(subscription: IntermediateRepresentation.OperationDefinition, intermediateRepresentation: IntermediateRepresentation, subscriptionSelections: SelectionSetVisitor.Operation) throws -> String {
```

Comment:
Spacing changes are intentional; I had the wrong Xcode indentation settings when I originally wrote this block.